Makefile

```diff
@@ -348,6 +361,73 @@ docker-images-only: docker-images-cassandra \
 	docker-images-tracegen \
 	docker-images-anonymizer
 
+.PHONY: multiarch-docker
```

Comment:
Couldn't we just move all this code into the script? Why do they need to be in the Makefile? I would restrict Makefile to building the binaries only, and then have all docker packaging etc. be done in the script.

Reply:
Yes, we could. I've moved all code related to building and uploading multi-arch docker images from Makefile to the scripts.